build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'com.github.hmcts'
-version '2.8.1'
+version '3.0.1'
```

Comment:
wouldn't say it's a major change for the data model version but depends on others I guess

Reply:
The existing one is v3.0.0 and I was just adding on top.